extensions/wikia/WikiFactory/WikiFactory.php

```diff
@@ -1253,18 +1256,19 @@ static public function getLocalEnvURL( $url, $forcedEnv = null ) {
 		// put the address back into shape and return
 		switch ( $environment ) {
```

Comment:
![MAJOR](https://raw.githubusercontent.com/SonarCommunity/sonar-github/master/images/severity-major.png) Add a "case default" clause to this "switch" statement. [![rule](https://raw.githubusercontent.com/SonarCommunity/sonar-github/master/images/rule.png)](https://sonarqube.wikia-inc.com/coding_rules#rule_key=php%3AS131)